RELEASE.md

```diff
@@ -0,0 +1,31 @@
+# XTF release process
+
+XTF repository is hosted on [Bintray](https://bintray.com/)
+
+## Pre-requisitions to create new XTF release
+* Write access to XTF upstream repository
+
+## Production release
+1. Clone upstream XTF repo, not a personal fork
+2. Maven Release plugin is pre-configured, release repository is defined in distribution management section, 
+GitHub repository in SCM section to push tags:
+
+```mvn release:clean release:prepare```
+
+in a nutshell it will create release tag, version and move to next devel version. This is interactive process and requires 
+few details to be provided, see the example:
+```
+What is the release version for "XTF"? (cz.xtf:utilities) 0.5: 0.5
+What is SCM release tag or label for "XTF"? (cz.xtf:utilities) utilities-0.5: : 0.5
+What is the new development version for "XTF"? (cz.xtf:utilities) 0.6-SNAPSHOT: 0.6-SNAPSHOT
+```
+
+finally tagged version is pushed to GitHub repository. GH Actions are configured to perform `mvn deploy` for new tags
+and push to maven repository after few minutes new version should appear in the Maven repository.
```

Comment:
So I don't need access to bintray in any way, right? The push is done by a GH actions service account.

Reply:
correct